content/2021-11-18-brigade-2-cuts-first-release-candidate.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Brigade 2 Cuts First Release Candidate"
+slug: "2021-11-18-brigade-2-cuts-first-release-candidate"
+description: "Brigade 2 has cut its first release candidate"
+date: "2021-11-18"
+author: "Kent Rancourt"
+summary: Brigade 2 has cut its first release candidate.
+---
+
+We have exciting news, Brigadiers!
+
+Yesterday afternoon, maintainers cut the first Brigade 2 release candidate. This release candidate (v2.0.0-rc.1) also marks Brigade 2's transition from serving the v2 beta API to serving the _stable_ v2 API. With this new level of stability, there's never been a better time to get started with Brigade 2, so please do consider heading over to our recently refreshed [QuickStart](https://v2--brigade-docs.netlify.app/intro/quickstart/) and taking us for a spin.
```

Comment:
Do you want to link to the actual release here?

Reply:
It seemed clunky to link to a chart, which is why I linked to the QuickStart.

Reply:
I was thinking at the actual GitHub release -- https://github.com/brigadecore/brigade/releases/tag/v2.0.0-rc.1

Reply:
Thought about it, but if you follow that, it's a dead end because you'll get the CLI and that's it. If we link to the QuickStart, it'll walk them through everything.

Reply:
Actually... I could drop a link to the QuickStart on the release page and that helps...

Fixing...

Reply:
Done